Dockerfile

```diff
@@ -44,5 +45,5 @@ RUN useradd -u 1000 -s /bin/bash mlserver && \
 USER 1000
 
 # Need to source `activate-env.sh` so that env changes get persisted
-CMD . ./hack/activate-env.sh $MLSERVER_ENV_TARBALL \
-    && mlserver start $MLSERVER_MODELS_DIR
+CMD . ./hack/activate-env.sh $MLSERVER_ENV_TARBALL $MLSERVER_VERSION \
+    && python -m mlserver.cli.main start $MLSERVER_MODELS_DIR
```

Comment:
this works but I am not sure if there are any caveats?

Reply:
Did you see any differences in using `python -m mlserver.cli.main` versus just doing `mlserver start`?